docs/tails_printing_guide.rst

```diff
@@ -48,8 +48,8 @@ Tails.
 Connect the printer to your Tails-booted computer via USB, then turn the printer
 on.
 
-Now, you'll want to single-click your way through **Applications** ▸
-**System Tools** ▸ **Settings** ▸ **Devices** ▸ **Printers**. The screenshot
+Now, you'll want to single-click your way through **Applications ▸
+**System Tools ▸ Settings ▸ Devices ▸ Printers**. The screenshot
```

Comment:
You've got an extra `**` at the beginning of line 52. 

It should be made clearer that this navigation starts at the system menu and then transitions into the settings applet.

Reply:
thanks, should be fixed now.